### DISCUSSION TITLE
max-width

### DISCUSSION THREAD

**Discussion by ghost** [codex-team/editor.js#1867] [Q&A] [answered]
How do we set max-width? 
It is 650px currently, i don't know why.

**Comment by joel611** [comment, marked as answer]
you can use css to override .ce-block__content and .ce-toolbar__content

**Comment by zerqs** [comment]
this is not working for me.
 have the css classes changed since Aug 2019?

**Comment by sunmachines** [comment]
Just open DOM debugger and find the classes you need to override.

**Comment by b-babic** [comment]
Like @joel611 said. unset the default styles from these two classes
```
.ce-block__content,
.ce-toolbar__content {
    max-width: unset;
}
```

**Comment by Rubix982** [comment]
How does this work exactly? I'm using NextJS, Material UI, and EditorJS together at the moment.

As expected, I get the class names,

![image](https://user-images.githubusercontent.com/41635766/140497079-6375824c-e0f7-47cd-8d78-84c82746d5d8.png)

I write CSS to over ride those,

![image](https://user-images.githubusercontent.com/41635766/140497180-63172e78-691e-4b36-9381-f7befefbcf57.png)

As expected, I see the CSS being properly loaded.

![image](https://user-images.githubusercontent.com/41635766/140497378-90c090b9-c140-44d1-b181-a7f464655173.png)

Do I see the CSS is actually updated? No.

![image](https://user-images.githubusercontent.com/41635766/140497492-be812325-3e63-48cf-84c5-a27391d68b6b.png)

I think my method is usually how CSS classes are over rided, no?

**Reply by emrecoban** [reply]
I have the same issue. Any solution? (Next.js with TailwindCSS)

**Comment by talyguryn** [comment]
same as #1891

**Comment by joshuavachon25** [comment]
I may have a solution. 
I am using SvelteKit (SSR) and I was scoping my style in my editorjs component. But if, instead, you add the css to the global style, it works! So, you probably scoped your style to your component, but instead copy it to the global css. 

I used this css to make it 100%:
`.ce-block__content, .ce-toolbar__content {
    max-width: calc(100% - 100px) !important;
}
.cdx-block {
    max-width: 100% !important;
}`

**Comment by gajda-w** [comment]
I know this is an old issue but is someone working with Nextjs and editorjs and have problem with max-width, just move css properties to global.css

`.ce-block__content,
.ce-toolbar__content, .codex-editor__redactor {
    max-width: unset;
}`

peace
